benchmark/LinearAsync.hs

```diff
@@ -104,5 +127,9 @@ main =
         -- Parallel stages in a pipeline
         , benchIO "parAppMap" Ops.parAppMap
         , benchIO "parAppSum" Ops.parAppSum
+        , benchMonadicSrcIO "concatMapWith (2,x/2)"
+            (Ops.concatStreamsWith parallel 2 Ops.value)
```

Comment:
It should be `Ops.value 'div' 2` acccording to the comment.

Reply:
Yes, changed, thanks!